unlock-app/src/components/creator/lock/CreatorLockSaved.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import styled from 'styled-components'
+import UnlockPropTypes from '../../../propTypes'
+import Icons from '../../interface/icons'
+
+export function CreatorLockSaved({ lock }) {
```

Comment:
`export default` here to avoid the line below and use the import consistently (`import {CreatorLockSaved} from './lock/CreatorLockSaved' above)

Reply:
Actually, keep the 2 lines, I think this is better but fix the import the above for consistency please ;)